src/Domain/Structure/ElementId.hpp

```diff
@@ -181,6 +181,15 @@ bool operator>=(const ElementId<VolumeDim>& lhs,
   return !(lhs < rhs);
 }
 
+/// Returns a bool if the element is the `0th` element in the domain.
```

Comment:
Please add some documentation of why this function exists, and that for any refined Domain, only the element containing the lower corner of Block0 will return `true`.  Also add a warning that the `ElementId` which will return `true` will change as the domain is refined